tests/integration/cli/test_batch.py

```diff
@@ -19,8 +20,10 @@ def test_batch_run(self):
         Tests executing a simple batch command to help catch regressions
         '''
         ret = 'Executing run on [{0}]'.format(repr('sub_minion'))
-
-        cmd = self.run_salt('\'*minion\' test.echo \'batch testing\' -b 50%')
+        if salt.utils.platform.is_windows():
```

Comment:
Why not use " for all platforms

Reply:
On unix like platforms the `*` could be globbed.
```
dwoz@carbon:~/test$ ls
meh.sh
dwoz@carbon:~/test$ cat meh.sh
#!/bin/sh

echo $@
dwoz@carbon:~/test$ ./meh.sh *minion
*minion
dwoz@carbon:~/test$ touch foominion
dwoz@carbon:~/test$ ./meh.sh *minion
foominion
```

Reply:
Good point.